src/layout/getstarted/page.js

```diff
@@ -34,6 +34,7 @@ const Page = ({
 <meta name="theme-color" content="#ffffff">
 <meta name="robots" content="index, follow">
 <meta name="author" content="Digital Transformation Agency">
+<meta name="description" content="Inclusive design, open-source code and shared insights">
```

Comment:
I wouldn't do this here man. I would recommend going into each folder in the `content/get-started` directory and adding a `description` as front matter to use in the meta tag. If we do this, it will mean that if we search something like "npm install design system" the google description under the link will be `Inclusive design, open-source code and shared insights` and it will be like that for every child page under t `/get-started`